src/common/exponent/exponentHelper.ts

```diff
@@ -4,6 +4,7 @@
 import * as path from "path";
 import * as Q from "q";
 import * as XDL from "./xdlInterface";
+import * as vscode from "vscode";
```

Comment:
You can't refer to the `vscode` package from this file as-is: Sources under `src/common` can be referenced both from within the vscode extension process and within the external debug adapter process, and the debug adapter process cannot access `vscode`.